src/main/java/jrr/by/lesson3/wordCounter/UniqueWordCounter.java

```diff
@@ -0,0 +1,78 @@
+package jrr.by.lesson3.wordCounter;
+
+import java.util.Map;
+
+import org.apache.log4j.Logger;
+
+public class UniqueWordCounter {
+
+    private Map<Integer, String> words;
+
```

Comment:
I think it is best to use HashMap <String, Integer> because HashMap contains unique keys